transformation/transformationPlugins/jsonMessageTransformers/jsonTypeMappingsSanitizationTransformer/src/test/java/org/opensearch/migrations/transform/TypeMappingsSanitizationDocBackfillTest.java

```diff
@@ -0,0 +1,43 @@
+package org.opensearch.migrations.transform;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+
+import org.opensearch.migrations.testutils.JsonNormalizer;
+
+import lombok.Lombok;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.ObjectMapper;
+
+@Slf4j
+public class TypeMappingsSanitizationDocBackfillTest {
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    @Test
+    public void test() throws Exception {
+        var testString = "{\n" +
+                "  \"index\": { \"_index\": \"performance\", \"_type\": \"network\", \"_id\": \"1\" },\n" +
+                "  \"source\": { \"field1\": \"value1\" }\n" +
+                "}";
+
+        var expectedString = "{\n" +
+            "  \"index\": { \"_index\": \"network\", \"_id\": \"1\" },\n" +
```

Comment:
Blocker

This was a bit surprising, assuming this is the default behavior.  I'd have expected us to have the transformed index name be `performance_network` based on prior conversation/context.  Curious to hear a bit more about the motivation of this design choice, which seems to erase a clear link to the original index and set us up for a bigger chance of collisions (multiple indices could have types with the same name, and only by prefixing the new index name with the original would you keep those documents from all ending up in the same index on the target).